packages/engine/Specs/Scene/GaussianSplatPrimitiveSpec.js

```diff
@@ -47,6 +47,102 @@ describe(
       ResourceCache.clearForSpecs();
     });
 
+    it("loads a Gaussian splats tileset", async function () {
+      const tileset = await Cesium3DTilesTester.loadTileset(
+        scene,
+        tilesetUrl,
+        options,
+      );
+      scene.camera.lookAt(
+        tileset.boundingSphere.center,
+        new HeadingPitchRange(0.0, -1.57, tileset.boundingSphere.radius),
+      );
+      expect(tileset.hasExtension("3DTILES_content_gltf")).toBe(true);
+      expect(tileset.isGltfExtensionUsed("KHR_gaussian_splatting")).toBe(true);
+      expect(
+        tileset.isGltfExtensionUsed("KHR_gaussian_splatting_compression_spz_2"),
+      ).toBe(true);
+      expect(tileset.isGltfExtensionRequired("KHR_gaussian_splatting")).toBe(
+        true,
+      );
+      expect(
+        tileset.isGltfExtensionRequired(
+          "KHR_gaussian_splatting_compression_spz_2",
+        ),
+      ).toBe(true);
+
+      const tile = await Cesium3DTilesTester.waitForTileContentReady(
+        scene,
+        tileset.root,
+      );
+
+      expect(tile.content).toBeDefined();
+      expect(tile.content instanceof GaussianSplat3DTileContent).toBe(true);
+    });
+
+    xit("loads a Gaussian splats tileset and toggles visibility", async function () {
+      const tileset = await Cesium3DTilesTester.loadTileset(
+        scene,
+        tilesetUrl,
+        options,
+      );
+      scene.camera.lookAt(
+        tileset.boundingSphere.center,
+        new HeadingPitchRange(0.0, -1.57, tileset.boundingSphere.radius),
+      );
+      const tile = await Cesium3DTilesTester.waitForTileContentReady(
+        scene,
+        tileset.root,
+      );
+      expect(tile.content).toBeDefined();
+      expect(tileset.gaussianSplatPrimitive).toBeDefined();
+      expect(scene).notToRender([0, 0, 0, 255]);
+
+      tileset.show = false;
+      expect(scene).toRender([0, 0, 0, 255]);
+    });
```

Comment:
Since this test isn't doing anything at the moment, let's remove this `xit`'d spec from the _legacy_ describe block as it may go away sooner.